src/styles/navstyles.css

```diff
@@ -16,11 +16,49 @@
   
   html,
   body {
-    font-family: "Segoe UI Variable Display", "Segoe UI", -apple-system, Roboto,
-      sans-serif, serif;
+    font-family: "Roboto Slab", serif;
     margin: 0;
+    font-weight: bold;
     /* overflow-x: hidden; */
   }
+
+  header {
```

Comment:
Did you have any issues making the padding even on the left and right for the header? I think I had this issue last semester as well if so. Let me know if I can be of assistance.
![image](https://github.com/user-attachments/assets/16e37e58-531d-4f8c-9eec-b4e53c109fd0)

Reply:
Second this! The navigation buttons should be aligned to the right hand side of the nav bar